.mocharc.json

```diff
@@ -0,0 +1,3 @@
+{
+    "ignore": ["test/fixtures/**/*"]
+}
```

Comment:
let's keep the mocha.opts file instead of making this a json

Reply:
OK, i understand. I has fixed